tools/pipelines-tasks/common/lib/nuget.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<!-- defaultPushSource key works like the 'defaultPushSource' key of NuGet.Config files. -->
+	<!-- This can be used by administrators to prevent accidental publishing of packages to nuget.org. -->
+	<packageRestore>
+		<!-- Allow NuGet to download missing packages -->
+		<add key="enabled" value="True" />
+
+		<!-- Automatically check for missing packages during build in Visual Studio -->
+		<add key="automatic" value="True" />
+	</packageRestore>
+	<!-- Default Package Sources; works like the 'packageSources' section of NuGet.Config files. -->
+	<!-- This collection cannot be deleted or modified but can be disabled/enabled by users. -->
+	<packageSources>
+		<clear />
+		<add key="AppStreaming.PackagingServices.Artifacts" value="./" />
```

Comment:
The key name can be changed to something more specific.

Reply:
Done. changed to AppAttachFramework.Artifacts